.github/workflows/release.yml

```
@@ -1,187 +1,193 @@
+          mv release-assets/gestalt-cli-windows/gestalt_cli.exe release-assets/gestalt-cli-windows.exe
+          # Package Linux binary as tar.gz (binary is already in gestalt-cli-linux/)
+          mv release-assets/gestalt-cli-linux/gestalt_cli release-assets/gestalt-cli-linux/gestalt
+          tar -czf release-assets/gestalt-cli-linux.tar.gz -C release-assets gestalt-cli-linux
+          # Package macOS binary as tar.gz
+          mv release-assets/gestalt-cli-macos/gestalt_cli release-assets/gestalt-cli-macos/gestalt
+          tar -czf release-assets/gestalt-cli-macos.tar.gz -C release-assets gestalt-cli-macos
+        env:
+          FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.11)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)